awx_collection/plugins/modules/tower_project.py

```diff
@@ -254,22 +271,14 @@ def main():
 
     # Extract our parameters
     name = module.params.get('name')
-    description = module.params.get('description')
```

Comment:
Why did a bunch of these lines go away?

Reply:
They moved to line 352 to only get them if the value is not None, Otherwise sometimes they would get overwritten on the copy command as it was set to None. 
With the way it was originally, if description was not supplied it would become "", because description returned None here.